pytest/lib/bridge.py

```diff
@@ -175,9 +175,9 @@ def cleanup(self):
                 self.stderr.close()
             self.kill()
         except BaseException:
-            print("Kill %s failed on cleanup!" % (self.__class__.__name__))
+            logger.info("Kill %s failed on cleanup!" % (self.__class__.__name__))
             traceback.print_exc()
```

Comment:
logger.error('Kill %s failed on cleanup', type(self).__name__, exc_info=sys.exc_info())

No need for `traceback.print_exc()` or the two empty lines.